app/views/custom/proposals/_geozone.html.erb

```diff
@@ -0,0 +1,5 @@
+<ul id="geozone" class="no-bullet geozone">
```

Comment:
I think the `id` attribute is not necessary and will actually result in invalid HTML since the element appears multiple times on the same page (once per proposal in the proposals index).